pkgs/applications/editors/spacevim/helptags.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/autoload/SpaceVim.vim b/autoload/SpaceVim.vim
+index 16688680..fcafd6f7 100644
+--- a/autoload/SpaceVim.vim
++++ b/autoload/SpaceVim.vim
+@@ -1255,13 +1255,6 @@ function! SpaceVim#end() abort
+     let &helplang = 'jp'
+   endif
+   ""
+-  " generate tags for SpaceVim
```

Comment:
Why this patch is needed? the tags should be generate when spacevim is installed.

Reply:
@wsdjeg -- The way in which Nix operates is that the applications are stored in _/nix/store_ which is a **read-only** directory structure that is CAS (content addressable storage).

SpaceVim I found wants to generate the helptags in the same directory as the SpaceVim git directory which is read-only & would fail (_spacevim_root_dir)

At the moment I've disabled the help tag generation but maybe a better solution would be to generate the helptag docs in global_dir (i.e. SPACEVIM_DIR) which is read+write

Reply:
> are stored in /nix/store which is a read-only directory structure that is CAS (content addressable storage).

While the reasoning about your patch is correct, I'd like to point out that `/nix/store` is actually not entirely content-addressable: this is true for [`fixed-output derivations`](https://nixos.org/nixos/nix-pills/nix-store-paths.html#idm140737319611824), but not for package-definitions since the hash is generated using the derivation's source.

Efforts towards more content-addressability (i.e. hashing a derivation's output) are proposed here: https://github.com/NixOS/rfcs/pull/62